### PR TITLE
An example of an OCD LUA Coder - Neth

### PR DESCRIPTION
**Pull request by MrUnknownGamer**
Since I'm bored and I have OCD, I decided to fix some of the TTT Code, will be pushing more in the next hour.

- beamup and beamdown effects: Removed unused vars; "mat_sparkle" &
"mid"
- C4 cl_init: Fixed Whitespace, removed unused var; beep: Line 7, ch &
cw: Line 44, wc: Line 260, and result, Line 470

More to come, don't commit yet

### CODE REVIEW

**Comment by Kefta** [comment]
You could have finished the changes then made the PR

**Comment by bmwalters** [comment]
Also a lot of this (whitespace cleanup) can be automated...

**Comment by thegrb93** [comment]
I don't think I've ever seen a whitespace restyle get pulled. All it does is increase file sizes anyway.

**Comment by MrUnknownGamer** [comment]
Was doing while u commented.

You better thank me for all of this guys :+1:

**Comment by bmwalters** [comment]
If this does get approved, you should probably squash those commits when you're done.

File: garrysmod/gamemodes/terrortown/gamemode/cl_disguise.lua [hunk 1]
```diff
@@ -42,7 +42,7 @@ function DISGUISE.Draw(client)
    surface.SetTextColor(255, 0, 0, 230)
 
    local text = trans("disg_hud")
-   local w, h = surface.GetTextSize(text)
+   local h = surface.GetTextSize(text)
```
Comment (bmwalters) [inline garrysmod/gamemodes/terrortown/gamemode/cl_disguise.lua]: and you fail!

File: garrysmod/gamemodes/terrortown/gamemode/cl_awards.lua [hunk 2]
```diff
@@ -742,12 +736,12 @@ local function TimeOfDeath(events, scores, players, traitors)
    local traitor_win = nil
 
    local e = nil
-   for i=#events, 1, -1 do
+   for i = #events, 1, -1 do
       e = events[i]
 
       if e.id == EVENT_FINISH then
          time_near_end = e.t - near
-         traitor_win = (e.win == WIN_TRAITOR)
```
Comment (bmwalters) [inline garrysmod/gamemodes/terrortown/gamemode/cl_awards.lua]: this syntax is acceptable IMO

File: garrysmod/gamemodes/terrortown/gamemode/cl_equip.lua [hunk 3]
```diff
@@ -469,7 +467,7 @@ local function ReceiveBought()
 
    ply.bought = {}
    local num = net.ReadUInt(8)
-   for i=1,num do
+   for i = 1,num do
```
Comment (bmwalters) [inline garrysmod/gamemodes/terrortown/gamemode/cl_equip.lua]: spaced out the `=` but not the `,`?

File: garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua [hunk 4]
```diff
@@ -151,7 +151,7 @@ function LANG.SetActiveLanguage(lang_name)
 end
 
 function LANG.Init()
-   local lang_name = GetConVarString("ttt_language")
+   local lang_name = GetConVar("ttt_language")
```
Comment (bmwalters) [inline garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua]: another fail

**Comment by bmwalters** [comment]
I just pointed out two error-causing segments then I got bored. There could be many things slipping through the cracks here; we should just leave code cleanup like this to automated tools.

File: garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua (hunk 4 above)
Comment (Kefta) [inline garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua]: Cache the GetConVar, THEN use convar:GetString().

File: garrysmod/gamemodes/terrortown/gamemode/cl_disguise.lua (hunk 1 above)
Comment (Kefta) [inline garrysmod/gamemodes/terrortown/gamemode/cl_disguise.lua]: If you want to mask the unused variable, just make it's name _

File: garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua (hunk 4 above)
Comment (robotboy655) [inline garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua]: GetConVarString already caches and uses GetConVar, the change is pointless.

**Comment by bmwalters** [comment]
oh man I just remembered, there is a tool that can do this! Falco's GLuaFixer!
Windows:
`type base_ammo_ttt.lua | glualint --pretty-print`
*nix:
`cat base_ammo_ttt.lua | glualint --pretty-print`

Then you just have to fix the indentation level.

**Comment by Kefta** [comment]
@bmwalters I believe Falco has a Linter or something that removes unused variables

**Comment by MrUnknownGamer** [comment]
Comment will be removed. 

I'm at dinner at NJ shore. I'll fix this when I get home or you can pull request on my fork.

**Comment by MrUnknownGamer** [comment]
Moved to here since I don't know how to use github correctly. https://github.com/garrynewman/garrysmod/pull/1214

**Comment by MrUnknownGamer** [comment]
ReOpening. Had sandbox and ttt in same pull request

File: garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua [hunk 5]
```diff
@@ -151,7 +151,7 @@ function LANG.SetActiveLanguage(lang_name)
 end
 
 function LANG.Init()
-   local lang_name = GetConVar("ttt_language")
+   local lang_name = GetConVar:GetString("ttt_language")
```
Comment (bmwalters) [inline garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua]: I would say you tried, but that's giving you too much credit 😉

File: garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua [hunk 6]
```diff
@@ -151,7 +151,7 @@ function LANG.SetActiveLanguage(lang_name)
 end
 
 function LANG.Init()
-   local lang_name = GetConVarString("ttt_language")
+   local lang_name = GetConVar:GetString("ttt_language")
```
Comment (bmwalters) [inline garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua]: *sigh*

File: garrysmod/gamemodes/terrortown/gamemode/cl_radar.lua [hunk 7]
```diff
@@ -203,7 +203,7 @@ function RADAR:Draw(client)
    surface.SetTextColor(255, 0, 0, 230)
 
    local text = GetPTranslation("radar_hud", {time = FormatTime(remaining, "%02i:%02i")})
-   local w, h = surface.GetTextSize(text)
+   local h = surface.GetTextSize(text)
```
Comment (bmwalters) [inline garrysmod/gamemodes/terrortown/gamemode/cl_radar.lua]: still have this bug

File: garrysmod/gamemodes/terrortown/gamemode/cl_disguise.lua [hunk 8]
```diff
@@ -1,51 +1,45 @@
...
-   local w, h = surface.GetTextSize(text)
-
-   surface.SetTextPos(36, ScrH() - 160 - h)
-   surface.DrawText(text)
+    if (not client) or (not client:IsActiveTraitor()) then return end
+    if not client:GetNWBool("disguised", false) then return end
+    surface.SetFont("TabLarge")
+    surface.SetTextColor(255, 0, 0, 230)
+    local text = trans("disg_hud")
+    local h = surface.GetTextSize(text)
```
Comment (bmwalters) [inline garrysmod/gamemodes/terrortown/gamemode/cl_disguise.lua]: anotha one

**Comment by bmwalters** [comment]
@MrUnknownGamer You don't know when to quit. I thought I made it clear that this is a job for automated tools - not you who is simply introducing bugs and changing so many files that other bugs will simply slip through.

File: garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua (hunk 6 above)
Comment (Kefta) [inline garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua]: FFS, read the documentation. Cache it OUTSIDE the function, then run GetString on the local

**Comment by MrUnknownGamer** [comment]
I don't see the problem, and I used the pretty print feature.

**Comment by bmwalters** [comment]
Scroll up. Three instances where the gamemode is broken by your changes. Those are just the ones I've skimmed through and found.

**Comment by Kefta** [comment]
@MrUnknownGamer You introduced bugs. Zerf labelled them for you

**Comment by Kefta** [comment]
@robotboy655 Please close this

**Comment by MrUnknownGamer** [comment]
Jesus, calm down boys. I'm a bit tired after my day at the shore and being up till 2am in the morning.

**Comment by Kefta** [comment]
Then don't make PRs in your state

**Comment by svdm** [comment]
This is a big bag of unnecessary fiddling mixed with wrong things and only a small sprinkling of actual improvements.

Feel free to maintain this in a `garrysmod-ocd` fork for those gmod players looking for the artisanal whitespace experience.

**Pull request closed without merging**